src/Evolution/Systems/Cce/GaugeTransformBoundaryData.cpp

```diff
@@ -163,7 +163,7 @@ void GaugeAdjustedBoundaryValue<Tags::BondiQ>::apply_impl(
                     0.25 *
                         square(evolution_gauge_j * conj(evolution_gauge_dy_j) +
                                evolution_gauge_dy_j * conj(evolution_gauge_j)) /
-                        square(evolution_gauge_k))) -
+                        square(evolution_gauge_k))) +
```

Comment:
same question, is there a test that could have caught this?

Reply:
Unfortunately, this one isn't easy to find with simple tests because it is still a valid coordinate transformation with the wrong sign, it's just not the coordinate transformation we want that gives an asymptotically inertial system. The real answer to 'how do we test this' is the bouncing black hole test, which is how I found the sign error, and will be a fairly near-future PR once we get the main executable in -- related is #1641.